jobs/service-fabrik-broker/templates/config/settings.yml.erb

```diff
@@ -30,11 +30,13 @@ production:
   log_level: <%= p('log_level') %>
   sys_log_level: <%= p('sys_log_level') %>
   enable_bosh_rate_limit: <%= p('enable_bosh_rate_limit') %>
+  enable_cross_organization_sharing: <%= p('enable_cross_organization_sharing') %>
```

Comment:
I think it would be needed in the BOSH operator job as well.

Reply:
Done now... The previous commit was swallowed up by a local git bug